controllers/API/comment.js

```diff
@@ -0,0 +1,97 @@
+var express = require('express');
+var router  = express.Router();
+var Comment = require('mongoose').model('comment');
+var Forum = require('mongoose').model('forum');
+
+
+router.post('/API/forum/comment',function (req, res) {
+    var newComment = new Comment({
+        text: req.body.text,
+        author: req.body.author,
+        posterID: req.body.posterID,
+        imgURL:req.body.imgURL
+    });
+    newComment.save(function (err, result) {
+        if (err) {
+            res.status(201);
+            res.send(err);
```

Comment:
Do we want to send a 201 in case of an error?

Reply:
According to the pdf charles added in slack. Lets discuss and agree on a uniform way of sending back errors today in the meeting.